integration-tests/kubernetes/src/it/kubernetes-build-and-deploy/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,22 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
```

Comment:
Why are these Dockerfiles in the test? I thought it was extracting the Dockferfile from the extension?

Reply:
They are not needed. I'll ditch them.